### PR TITLE
:sparkles: Add translation for write_own_pipeline.mdx

### PR DESCRIPTION
**Pull request by cuchoco**

### CODE REVIEW

**Comment by cuchoco** [comment]
timstep: `timestep` array 가 계속 등장해서 타임스텝 한국어 대신 timstep으로 표기했습니다.
noise residual, noisy residual: 노이즈 잔여량이라고 번역되었는데 일단은 영어 원문 그대로 표기했습니다.
diffusion: 디퓨전이라고 한국어로 적는게 더 나을까요?

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# 파이프라인, 모델 및 스케줄러 이해하기
+
+[[colab에서 열기]]
+
+🧨 Diffusers는 사용자 친화적이며 유연한 도구 상자로, 사용사례에 맞게 diffusion 시스템을 구축 할 수 있도록 설계되었습니다. 이 도구 상자의 핵심은 모델과 스케줄러입니다.  [`DiffusionPipeline`]은 평의를 위해 이러한 구성 요소를 번들로 제공하지만, 파이프라인을 분리하고 모델과 스케줄러를 개별적으로 사용해 새로운 diffusion 시스템을 만들 수도 있습니다.
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: 평의 -> 편의

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+1. 모델과 스케줄러를 불러옵니다:
+
+    ```py
+    >>> from diffusers import DDPMScheduler, UNet2DModel
+
+    >>> scheduler = DDPMScheduler.from_pretrained("google/ddpm-cat-256")
+    >>> model = UNet2DModel.from_pretrained("google/ddpm-cat-256").to("cuda")
+    ```
+
+2. 노이즈 제거 프로세스를 실행할 timestep수를 설정합니다:
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: timestep 수 띄어쓰기

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    >>> from PIL import Image
+    >>> import numpy as np
+
+    >>> image = (input / 2 + 0.5).clamp(0, 1)
+    >>> image = image.cpu().permute(0, 2, 3, 1).numpy()[0]
+    >>> image = Image.fromarray((image * 255).round().astype("uint8"))
+    >>> image
+    ```
+
+다음 섹션에서는 여러분의 기술을 시험해보고 좀 더 복잡한 Stable Diffusion 파이프라인을 분석해 보겠습니다. 방법은 거의 동일합니다. 필요한 구성요소들을 초기화하고 timestep수를 설정하여 `timestep`배열을 생성합니다. 노이즈 제거 루프에서 `timestep`배열이 사용되며, 이 배열의 각 요소에 대해 모델은 노이즈가 적은 이미지를 예측합니다. 노이즈 제거 루프는 `timestep`을 반복하고 각 timestep에서 nosie residual을 출력하고 스케줄러는 이를 사용하여 이전 timestep에서 노이즈가 덜한 이미지를 예측합니다. 이 프로세스는 `timestep`배열의 끝에 도달할 때까지 반복됩니다.
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: nosie -> noisy

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+
+한번 사용해 봅시다!
+
+## Stable Diffusion 파이프라인 해체하기
+
+Stable Diffusion 은 텍스트-이미지 *latent diffusion* 모델입니다. latent diffusion 모델이라고 불리는 이유는 실제 픽셀 공간 대신 이미지의 저차원의 표현으로 작업하기 때문이고, 메모리 효율이 더 높습니다. 인코더는 이미지를 더 작은 표현으로 압축하고, 디코더는 압축된 표현을 다시 이미지로 변환합니다. 텍스트-이미지 모델의 경우 텍스트 임베딩을 생성하기 위해 tokenizer와 인코더가 필요합니다. 이전 예제에서 이미 UNet 모델과 스케줄러가 필요하다는것은 알고 계셨을 것입니다.
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: 텍스트-이미지 -> 영어로
필요하다는 것은

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+<Tip>
+
+💡 `guidance_scale`매개변수는 이미지를 생성할 때 프롬프트에 얼마나 많은 가중치를 부여할지 결정합니다.
+
+</Tip>
+
+다른 프롬프트를 생성하고 싶다면 원하는 프롬프트를 자유롭게 선택하세요!
+
+```py
+>>> prompt = ["a photograph of an astronaut riding a horse"]
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx:163]: 주석도 가능하면 번역합니다!

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+```py
+>>> text_input = tokenizer(
+...     prompt, padding="max_length", max_length=tokenizer.model_max_length, truncation=True, return_tensors="pt"
+... )
+
+>>> with torch.no_grad():
+...     text_embeddings = text_encoder(text_input.input_ids.to(torch_device))[0]
+```
+
+또한 패딩 토큰의 임베딩인 *비조건부 텍스트 임베딩*을 생성해야 합니다. 이 임베딩은 조건부 `text_embeddings`과 동일한 모양(`batch_size` 그리고 `seq_length`)을 가져야 합니다:
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: unconditional

Comment (tjdtnsu) [reply docs/source/ko/using-diffusers/write_own_pipeline.mdx]: shape

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+
+```py
+>>> max_length = text_input.input_ids.shape[-1]
+>>> uncond_input = tokenizer([""] * batch_size, padding="max_length", max_length=max_length, return_tensors="pt")
+>>> uncond_embeddings = text_encoder(uncond_input.input_ids.to(torch_device))[0]
+```
+
+두번의 포워드 패스를 피하기 위해 조건부 임베딩과 비조건부 임베딩을 배치로 연결하겠습니다:
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: forward pass

Comment (tjdtnsu) [reply docs/source/ko/using-diffusers/write_own_pipeline.mdx]: 배치(batch)

File: docs/source/ko/using-diffusers/write_own_pipeline.mdx [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/blog/assets/98_stable_diffusion/stable_diffusion_k_lms.png"/>
+</div>
+
+## 다음 단계
+
+기본 파이프라인부터 복잡한 파이프라인까지, 자신만의 diffusion 시스템을 작성하는 데 필요한 것은 노이즈 제거 루프뿐이라는 것을 알 수 있었습니다. 이 루프는 스케줄러의 timesteps를 설정하고, 이를 반복하며, UNet 모델을 호출하여 noise residual을 예측하고 스케줄러에 전달하여 이전 노이즈 샘플을 계산하는 과정을 번갈아 가며 수행해야 합니다.
+
+이것이 바로 🧨 Diffusers가 설계된 목적입니다: 모델과 스케줄러를 사용해 자신만의 디퓨전 시스템을 직관적이고 쉽게 작성할 수 있도록 하기 위해서입니다.
```
Comment (tjdtnsu) [inline docs/source/ko/using-diffusers/write_own_pipeline.mdx]: 디퓨전 -> diffusion

**Review by tjdtnsu — CHANGES_REQUESTED** [review]

**Review by tjdtnsu — APPROVED** [review]
